qa/SidechainTestFramework/account/smart_contract_resources/contracts/ForgerStakes.sol

```diff
@@ -32,9 +32,12 @@ interface ForgerStakes {
     function openStakeForgerList(uint32 forgerIndex, bytes32 signature1, bytes32 signature2) external returns (bytes memory);
     
     function stakeOf(address owner) external view returns (uint256);
-    
+
+    function getAllForgersStakesOfUser(address owner) external view returns (StakeInfo[] memory);
```

Comment:
this method getAllForgersStakesOfUser is no more present
(we replaced it with the paginated one getPagedForgersStakesByUser below)